core/src/main/java/org/apache/struts2/interceptor/debugging/DebuggingInterceptor.java

```diff
@@ -210,24 +215,24 @@ public void beforeResult(ActionInvocation inv, String actionResult) {
                         public void beforeResult(ActionInvocation inv, String actionResult) {
                             String rootObjectExpression = getParameter(OBJECT_PARAM);
                             if (rootObjectExpression == null)
-                                rootObjectExpression = "#context";
+                                rootObjectExpression = "top";
```

Comment:
Hi.  The change to `top` seems OK, but is there a risk it might break anything (vs `#context`) ?

Also, would adding `{ }` to the if block here as well as https://github.com/apache/struts/blob/1a0ef45160eaf3e66d828dfa363a314e690b0249/core/src/main/java/org/apache/struts2/interceptor/debugging/DebuggingInterceptor.java#L233 be something to consider as well ?

Reply:
Access to `#context` was removed some times ago, I switched to `action` which is a better choice. And added `{ }` :)